packages/utilities/src/array.ts

```diff
@@ -70,3 +70,12 @@ export function toMatrix<T>(items: T[], columnCount: number): T[][] {
     return rows;
   }, [] as T[][]);
 }
+
+/**
+ * Given an array, it returns a new array that does not contain the item at the given index.
+ * @param array - The array to operate on
+ * @param index - The index of the element to remove
+ */
+export function removeElement<T>(array: T[], index: number): T[] {
+  return [...array.slice(0, index), ...array.slice(index + 1)];
```

Comment:
I was curious if this method was faster than simply slicing and splicing, turns out it's slower.

https://codepen.io/dzearing/pen/zdLEZg?editors=0011

You should probably just `slice()` and then `splice(index, 1)`